CONTRIBUTING.md

```diff
@@ -160,6 +160,44 @@ const MyComponent: FC<MyComponentProps> = ({ prop1, prop2... }) => (
 export default MyComponent;
 ```
 
+#### How to I18n my component
```

Comment:
I don't think this content should be on the contributing guidelines.

I also don't think we should tell users how to use `react-intl`. We already forward potential contributors to the react-intl docs.